test/index.js

```diff
@@ -222,6 +227,7 @@ describe('Datastore', function() {
             scopes: v1.DatastoreClient.scopes,
             servicePath: datastore.baseUrl_,
             port: 443,
+            projectId: undefined,
```

Comment:
This test will fail if the environment has DATASTORE_PROJECT_ID defined for any reason (e.g. in the user profile). Can we explicitly delete it for this test? Also, I would add a test that would verify that the value from DATASTORE_PROJECT_ID comes to datastore.options (if there is no such test).

Reply:
Good call, done.